gocron.go

```diff
@@ -96,29 +95,27 @@ func Jobs() []*Job {
 }
 
 func formatTime(t string) (hour, min, sec int, err error) {
-	var er = errors.New("time format error")
 	ts := strings.Split(t, ":")
 	if len(ts) < 2 || len(ts) > 3 {
-		err = er
-		return
+		return 0, 0, 0, ErrTimeFormat
```

Comment:
this will look leaner on a next PR where we will return a struct instead of bare values